index.html

```diff
@@ -1572,6 +1589,66 @@
 
+## Support Choosing Which Information to Present {#support-choosing-info}
+
+<div class="practice">
+  <span class="practicelab" id="principle-support-choosing-info">
+    [=User agents=] should support [=people=] in choosing which information they provide to [=parties=] that
+    request it, up to and including allowing users to provide arbitrary information.
+  </span>
+</div>
+
+[=Parties=] can invest time and energy into automating ways of gathering [=data=] from [=people=] and can
+design their products in ways that make it a lot easier for [=people=] to disclose information than not, whereas
+[=people=] typically have to manually wade through options, repeated prompts, and deceptive patterns. In many
+cases, the absence of data — when a [=person=] refuses to provide some information — can also be identifying
+or revealing. Additionally, APIs can be defined or implemented in rigid ways that can prevent people from
+accessing useful functionality. For example, I might want to look for restaurants in a city I will be visiting
+this weekend, but if my geolocation is forcefully set to match my GPS, a restaurant-finding
+site might only allow searches in my current location.
+
+[=User agents=] should make it simple for [=people=] to [present the identity they wish
+to](#principle-identity-per-context) and to provide information about themselves or their devices in
+ways that they control. This empowers [=people=] to live in obscurity ([[?Lost-In-Crowd]],
+[[?Obscurity-By-Design]]) by rendering the provision of information according to their own
+preferences as easy as possible ([[?Obfuscation]]).
+
+<div class="practice">
+  <span class="practicelab" id="principle-no-facts-or-promises">
+    APIs should be designed such that data returned through an API does not assert a fact or make a
+    promise on the user's behalf about the user or their environment.
+  </span>
+</div>
+
+Instead, the API could indicate a [=person=]'s preference, a [=person=]'s chosen identity, a
+[=person=]'s query or interest, or a [=person=]'s selected communication style.
+
+<div class="example">
+For example, a [=user agent=] might support this principle by:
+
+* Generating domain-specific email addresses or other directed identifiers so that [=people=] can
+  log into the site without becoming recognisable across contexts.
+* Offering the option to generate geolocation and accelerometry data with parameters specified by
+  the [=user=].
+* Uploading a stored video stream in response to a camera prompt.
+* Automatically granting or denying permission prompts based on user configuration.
+* Filtering outbound network traffic so as to replace specific strings (hashed email, navigation
+  identifiers, consent strings).
```

Comment:
This doesn't seem connected to the "APIs should be designed" principle, but it is connected to the "support people in choosing" principle. What if we move the two principles to be together at the top of the section, and then the rest of the text can be about both together?

Reply:
I also found this example a little confusing, so I think it would be fine to leave out.

I don't think the principles need to be headings that all following text is directly part of. That sites should plan for user choice and including deception then leads to a variety of examples.